examples/vars.tf

```diff
@@ -0,0 +1,19 @@
+variable "AWS_ACCESS_KEY" {
+}
+
+variable "AWS_SECRET_KEY" {
+}
+
+variable "AWS_REGION" {
+  default = "ap-south-1"
+}
+
+variable "AMIS" {
+  type = map(string)
+  default = {
+    ap-south-1 = "ami-0860c9429baba6ad2"
+    ap-south-1 = "ami-0860c9429baba6ad2"
+    ap-south-1 = "ami-0860c9429baba6ad2"
```

Comment:
is this section duplictaed? @imnitin28

Reply:
@surajsaini95 
Yes, I could have used different regions or just one.
Resolved it by removing duplicate lines.